pycqed/analysis/measurement_analysis.py

```diff
@@ -30,7 +30,7 @@
 
 try:
     import qutip as qtp
-    from pycqed.analysis.tomography import Tomo_Multiplexed
+    #from pycqed.analysis.tomography import Tomo_Multiplexed
```

Comment:
sorry for this change, is not related to the pull request, but why is the line here anyway? I would expect the tomo to be imported separately by a user